src/App.js

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import FrontScreen from './pages/FrontScreen';
+import Categories from './pages/Categories';
+import NotFound from './pages/NotFound';
+
+import NavBar from './components/navBar/NavBar';
+
+const App = () => (
+  <BrowserRouter>
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<FrontScreen />} />
+      <Route path="/categories" element={<Categories />} />
```

Comment:
Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 😎